examples/with-custom-config/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "main": "index.js",
+  "scripts": {
+    "start": "webpack-dev-server"
```

Comment:
With regard to the code, is this example based off the `master` branch? 

[Our current examples](https://github.com/pburtchaell/redux-promise-middleware/tree/master/examples) use Parcel because it is more simple to setup and easier maintain over time. See issue #216 for the reason why and see PR #217 for the implementation of Parcel.

There's two things I think we could do here. 

1. Remove the "custom config" and "default config" examples (I think the docs are enough)
2. Update the examples to use Parcel

What do you think?